src/irr/asset/COBJMeshFileLoader.cpp

```diff
@@ -39,13 +39,361 @@ namespace std
 }
 */
 
-#define NEW_SHADERS
+namespace
+{
+    constexpr const char* VERT_SHADER_NO_UV = 
+R"(#version 430 core
+
+layout (location = 0) in vec3 vPos;
+layout (location = 3) in vec3 vNormal;
```

Comment:
wrap this in a `#ifndef + #define` guard for `VERTEX_INPUTS`

Reply:
like we talked on discord (there's a few examples with correct syntax)

Reply:
Actually examples are here https://github.com/buildaworldnet/IrrlichtBAW/issues/384#issuecomment-544130414